src/main/java/frc/robot/Robot.java

```diff
@@ -156,26 +169,46 @@ public void testPeriodic() {
   }
 
   @Config(name = "Indexer Speed", defaultValueNumeric = 0.3)
-  public void setIndexerSpeed(double indexerSpeed) {
+  public void setIndexerSpeed(final double indexerSpeed) {
     this.indexerSpeed = indexerSpeed;
     this.indexerController.setSpeed(this.indexerSpeed);
   }
 
   @Config(tabName = "Autonomous", name = "Distance", defaultValueNumeric = 36)
-  public void setAutonStraightDistance(double distance) {
+  public void setAutonStraightDistance(final double distance) {
     this.distance = distance;
     this.autonomousCommand.setDistance(this.distance);
   }
 
   @Config(tabName = "Autonomous", name = "Maximum Speed", defaultValueNumeric = .75)
-  public void setAutonMaxSpeedForDriveStraight(double maxSpeed) {
+  public void setAutonMaxSpeedForDriveStraight(final double maxSpeed) {
     this.maxSpeed = maxSpeed;
     this.autonomousCommand.setMaxSpeed(this.maxSpeed);
   }
 
   @Config(name = "Constant", defaultValueNumeric = .1)
-  public void setDriveStraightPValue(double pValue) {
+  public void setDriveStraightPValue(final double pValue) {
     this.pValue = pValue;
     this.autonomousCommand.setPValue(this.pValue);
   }
-}
+
+  public void disableCurvature() {
+
+    m_teleopSelected = m_teleopChooser.getSelected();
+    System.out.println("teleop selected " + m_teleopSelected);
```

Comment:
Remove this print (I know I added it, not you)

Reply:
done